app/helpers/curation_concerns/charts_helper.rb

```diff
@@ -1,7 +1,17 @@
 module CurationConcerns
   module ChartsHelper
-    def hash_to_flot(data)
-      data.map { |key, value| { label: key, data: value } }
+    def hash_to_chart(data)
```

Comment:
@tpendragon This seems a little more complex hash now that we include.  Could you add a comment with an example to show how to put in drill down information.